src/apis/getItems.ts

```diff
@@ -1,13 +1,15 @@
-import { API_KEY, QUERY_TYPE } from '@/constants/constantValues';
+import { API_BASE_URL, API_KEY, QUERY_TYPE } from '@/constants/constantValues';
+import getData from '@/helpers/getData';
+import { ContentType, IItems, QueryType } from '@/types/types';
 
-export const getItems = (queryType: string, contentType: string, search: string) => {
+const getItems = (queryType: QueryType, contentType: ContentType, search: string): Promise<IItems> => {
   const QUERY_TYPE_INFO = {
-    [QUERY_TYPE.TOP_RATED]: `https://api.themoviedb.org/3/${contentType}/top_rated?api_key=${API_KEY}&language=en-US`,
-    [QUERY_TYPE.SEARCH]: `https://api.themoviedb.org/3/search/${contentType}?api_key=${API_KEY}&language=en-US&query=${search}`,
+    [QUERY_TYPE.TOP_RATED]: `${API_BASE_URL}${contentType}/top_rated?api_key=${API_KEY}&language=en-US`,
+    [QUERY_TYPE.SEARCH]: `${API_BASE_URL}search/${contentType}?api_key=${API_KEY}&language=en-US&query=${search}`,
   };
+  const items = getData<IItems>(QUERY_TYPE_INFO[queryType]);
```

Comment:
```ts
const items = await getData<IItems>(QUERY_TYPE_INFO[queryType]);

return items
```
or
```ts
return getData<IItems>(QUERY_TYPE_INFO[queryType]);
```

Reply:
Should ContentType be connected to `IItems`?

Reply:
ContentType is important just to know what to request, movies or tv shows.